ocs_ci/ocs/cluster.py

```diff
@@ -590,25 +590,45 @@ def get_throughput_percentage(self):
         logging.info(f"The throughput percentage of the cluster is {throughput_percentage}%")
         return throughput_percentage
 
-    def get_rebalance_status(self):
+    def get_rebalance_status_base(self):
```

Comment:
if you are uploading one more commit, I suggest reverting this method name back as 'get_rebalance_status' name is not in use now